standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -213,6 +213,19 @@
   //todo: make these like OperationType and remove above char constatns
   enum TxnStatus {OPEN, ABORTED, COMMITTED, UNKNOWN}
 
+  public enum TxnType {
+    REPL_CREATED(0), READ_ONLY(1), UNKNOWN(2);
+
+    private final int value;
+    TxnType(int value) {
+      this.value = value;
+    }
+
+    public int getValue() {
```

Comment:
Won't by default integer values assigned to each enum member? Shall we use it instead of explicit assignment?

Reply:
ok